ddtrace/contrib/falcon/patch.py

```diff
@@ -23,7 +23,7 @@ def patch():
 def traced_init(wrapped, instance, args, kwargs):
     mw = kwargs.pop('middleware', [])
     service = os.environ.get('DATADOG_SERVICE_NAME') or 'falcon'
```

Comment:
With the updated `get_env()` we can do this to support both `DATADOG_SERVICE_NAME` and `DD_SERVICE_NAME` we we adopt the latter for consistency across environment variables:

```suggestion
    service = get_env("service_name", default="falcon")
```

Reply:
I mostly wanted to adjust any current usage of it, rather than focus on updating every place that fetches from environment.

wanted to keep the scope down a bit